config/routes.js

```diff
@@ -89,9 +89,7 @@ module.exports.routes = {
   '/game/chat': 'game/chat',
   '/game/gameData': 'game/game-data',
   // DEVELOPMENT ONLY
-  '/game/stackDeck': 'game/stack-deck',
-  '/game/loadFixture': 'game/load-fixture',
-  '/game/clear': 'game/clear-game',
```

Comment:
These routes are still needed except they should be moved down into the testing utilities section below and the right hand side of each should be updated to refer to the TestController instead of the /game folder.

Otherwise these urls won't resolve and requests to them will all 404